prefect_dbt/cli/commands.py

```diff
@@ -154,6 +154,9 @@ def trigger_dbt_cli_command_flow():
     # fix up empty shell_run_command_kwargs
     shell_run_command_kwargs = shell_run_command_kwargs or {}
 
+    # on Windows use PowerShell; else use bash
+    shell = "powershell" if sys.platform == "win32" else "bash"
+
     logger.info(f"Running dbt command: {command}")
-    result = await shell_run_command.fn(command=command, **shell_run_command_kwargs)
+    result = await shell_run_command.fn(command=command, shell=shell, **shell_run_command_kwargs)
```

Comment:
I like the idea of defaulting to powershell when running on a Windows machine, but we could get an error when users specify `shell` in `shell_run_command_kwargs`.

My recommendation would be to set `shell` within `shell_run_command_kwargs` if a value hasn't be provided by the user. If the user has provided `shell` within `shell_run_command_kwargs`, then we should respect that. Otherwise, we can reasonably default to powershell for windows users.

Reply:
Much more elegant!  Reflected in newest commit.